src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -707,7 +707,7 @@ uint64_t BlockchainLMDB::get_estimated_batch_size(uint64_t batch_num_blocks, uin
       ++num_blocks_used;
     }
     if (my_rtxn) block_rtxn_stop();
-    avg_block_size = total_block_size / num_blocks_used;
```

Comment:
Should `num_blocks_used` be initialized to `1` instead of `0`?

Reply:
If you reset it before the loop, I guess you could.